src/controllers/expensesController.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const expenseServices = require('../services/expenseServices');
+
+const getAll = (req, res) => {
+  const expenses = expenseServices.getAllExpenses(req.query);
+
+  res.status(200).send(expenses);
+};
+
+const createExpense = (req, res) => {
+  const isValidRequest = expenseServices.validateData(req.body);
+
+  if (!isValidRequest) {
+    res.status(400).send({ message: 'Validation error' });
+
+    return;
+  }
+
+  const newExpense = expenseServices.create(req.body);
+
+  res.status(201).send(newExpense);
+};
+
+const getOneExpense = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseServices.getExpenseById(+expenseId);
```

Comment:
better to convert values explicitly, like `Number(expenseId)`. Such type conversions are easier to read